README.md

```diff
@@ -1,18 +1,8 @@
-# Terraform Provider Scaffolding (Terraform Plugin Framework)
+# AEP Terraform Provider
 
-_This template repository is built on the [Terraform Plugin Framework](https://github.com/hashicorp/terraform-plugin-framework). The template repository built on the [Terraform Plugin SDK](https://github.com/hashicorp/terraform-plugin-sdk) can be found at [terraform-provider-scaffolding](https://github.com/hashicorp/terraform-provider-scaffolding). See [Which SDK Should I Use?](https://developer.hashicorp.com/terraform/plugin/framework-benefits) in the Terraform documentation for additional information._
+The AEP Terraform Provider generates a run-time Terraform provider for use with AEP-compliant APIs.
```

Comment:
This reads a bit like a tautology at the moment... what if you swapped the period for a `, so that...` and then completed the sentence? Perhaps also adding a real-world example would help orient people?